build-docker/action.yaml

```diff
@@ -96,10 +112,13 @@ runs:
           VERSION=${{ inputs.image-version }}
           REVISION=${{ inputs.image-version }}
           BRANCH=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.branch'] }}
```

Comment:
I think you are missing adding the `build-args` input here

Reply:
Not sure if the github diff makes it weird, but I have added it 🙈 

![image](https://github.com/user-attachments/assets/fb619313-3727-4930-a2b6-c6b862c7e7c4)

Reply:
Tested this indirectly through another action [here](https://github.com/turo/automatic-pricing-msvc/pull/1131/files)

Reply:
Oh It's been a long day I guess, sorry